.github/workflows/macos-installer.yml

```diff
@@ -44,11 +44,11 @@ jobs:
           path: "./out/macos-universal/crc-macos-installer.pkg"
       - name: Install crc pkg
         run: sudo installer -pkg out/macos-universal/crc-macos-installer.pkg -target /
-      - name: Set podman preset as part of config
-        run: crc config set preset podman
+      - name: Set microshift preset as part of config
```

Comment:
Running the microshift preset might result in a very diferent test, so best to let @jsliacan and @adrianriobo chime in on this.